modules/sdk-coin-xlm/src/xlm.ts

```diff
@@ -1112,29 +1107,9 @@ export class Xlm extends BaseCoin {
     return true;
   }
 
-  /**
-   * Derive a hardened child public key from a master key seed using an additional seed for randomness.
-   *
-   * Due to technical differences between keypairs on the ed25519 curve and the secp256k1 curve,
-   * only hardened private key derivation is supported.
-   *
-   * @param key seed for the master key. Note: Not the public key or encoded private key. This is the raw seed.
-   * @param entropySeed random seed which is hashed to generate the derivation path
-   */
-  deriveKeyWithSeed({ key, seed }: { key: string; seed: string }): { derivationPath: string; key: string } {
-    const derivationPathInput = utxolib.crypto.hash256(Buffer.from(seed, 'utf8')).toString('hex');
-    const derivationPathParts = [
-      999999,
-      parseInt(derivationPathInput.slice(0, 7), 16),
-      parseInt(derivationPathInput.slice(7, 14), 16),
-    ];
-    const derivationPath = 'm/' + derivationPathParts.map((part) => `${part}'`).join('/');
-    const derivedKey = Ed25519KeyDeriver.derivePath(derivationPath, key).key;
-    const keypair = stellar.Keypair.fromRawEd25519Seed(derivedKey);
-    return {
-      key: keypair.publicKey(),
-      derivationPath,
-    };
+  /** inheritdoc */
+  deriveKeyWithSeed(): { derivationPath: string; key: string } {
+    throw new NotSupported('method deriveKeyWithSeed not supported for eddsa curve');
```

Comment:
Was this wrongly implemented previously?

Reply:
unhardened derivation(from pub) wont give you the same pub as hardened derivation (from prv)